ecs-init/config/common_test.go

```diff
@@ -82,3 +82,36 @@ func TestGetAgentPartitionBucketRegion(t *testing.T) {
 			})
 	}
 }
+
+const EnvCgroupMountpoint = "CGROUP_MOUNTPOINT"
```

Comment:
nit: this can be `envCgroupMountpoint`

Reply:
Whoops, I meant for it to be using the const in common.go